apps/web/src/components/search/index.js

```diff
@@ -20,16 +20,29 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 import * as Icon from "../icons";
 import "./search.css";
 import Field from "../field";
+import { useStore } from "../../stores/theme-store";
 
 function SearchBox(props) {
+  const { theme } = useStore();
   return (
     <Field
       data-test-id="search-input"
       autoFocus
       id="search"
       name="search"
       type="text"
-      sx={{ m: 0, mx: 1, mt: 1 }}
+      sx={{
+        m: 0,
+        mx: 1,
+        mt: 1,
+        ...(theme === "dark" && {
+          "input:-webkit-autofill": {
+            WebkitTextFillColor: "rgba(231, 225, 225, 0.842)",
+            caretColor: "white",
+            fontSize: "inherit"
+          }
+        })
```

Comment:
Now let's get rid of applying this only for dark theme by using colors from the current theme.

You can see all theme colors here: https://github.com/streetwriters/notesnook/tree/master/packages/theme/src/theme/colorscheme

Reply:
OK